### ISSUE TITLE
Aliases are incorrectly registered as commands on the server

### ISSUE THREAD

**Issue by sasial-dev** [paradoxuum/commander#33]
Aliases are incorrectly registered as commands on the server. This leads to them being sent down within the `syncDispatch` event as commands. 

This can lead to cosmetic side effects such in the interface:
![RobloxStudio-Mz67U84K@2x](https://github.com/paradoxuum/commander/assets/44125644/faa35a81-ab91-4bb9-a4f7-58f7efc0ea4c)

It leads to a lot of warnings
![RobloxStudio-ZVzW96ZM@2x](https://github.com/paradoxuum/commander/assets/44125644/ef22f95c-4ee6-426f-a034-d426b6d0de70)

But it can also lead to errors where Commander thinks that the command already exists. Once this error occurs in the registry, the interface does not start up.
![RobloxStudio-QAmLCRsJ@2x](https://github.com/paradoxuum/commander/assets/44125644/0f459501-e2b0-412b-9933-477e708d6ee6)

(where `rnsignal` is an alias of `railnetSignalDebug`)

**Comment by paradoxuum** [comment]
The error itself has been fixed in commit [c254ef7](https://github.com/paradoxuum/commander/commit/c254ef7c6d2ef3cdfd8b136e560b482630cf0d07), but there is currently a bug in the interface where command aliases will show up in suggestions as the "primary" name. So if you have an alias to one command you'll have 2 suggestions with the same name. I've opened an issue for this - #34.

Aliases are intended to be registered as commands, i.e. their paths are added to the `commands` map, but they point to the same `BaseCommand` class. Currently this works fine with the fix above, but this may be changed if it causes issues when attempting to fix the suggestions issue.
